### PR TITLE
added new sql quiz questions

### PR DESCRIPTION
**Pull request by wheeleruniverse**
## Summary of changes

#17 

* contributed new SQL questions for use in the FCC Developer Quiz

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

### CODE REVIEW

File: src/data/sql-quiz.ts [hunk 1]
```diff
@@ -829,6 +829,28 @@ const sqlQuiz = [
     Explanation:
       "The BETWEEN operator is an intuitive way to use the WHERE clause to help narrow down a result set.",
     Link: "https://www.freecodecamp.org/news/a-beginners-guide-to-sql/"
+  },
+  {
+    Question:
+      "What SQL statement contains a valid subquery?",
+    Answer: "SELECT * FROM employees WHERE wage > (SELECT AVG(wage) FROM employees)",
+    Distractor1: "SELECT MAX(*) FROM employees",
+    Distractor2: "SELECT AVG(wage) FROM employees WHERE name IN ('name1', 'name2', 'name3')",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:838]: this looks valid to me
we should alter this one to be one of the wrong answers

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/sql-quiz.ts (hunk 1 above)
Comment (wheeleruniverse) [reply src/data/sql-quiz.ts:838]: Does that make it a good distractor? An in-clause is not a sub query, but I can still update this question if you would like me too!

Comment (jdwilkin4) [reply src/data/sql-quiz.ts:838]: yeah, I think at first glance it looked valid but at second look that makes sense why it isn't.
we could leave it.

Some questions, will have more obviously wrong and right answers why others will take a few more seconds to look at to check for the right answer.

So it is good to have a mixture of both 👍

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**